### PR TITLE
feat: Ellison p4 HW emulator

### LINKED ISSUE #1074: feat: Ellison p4 HW emulator
Let's make this pattern:
image

Please work at this file: https://github.com/TroyTae/game-of-life/blob/master/src/life/oscillator/ellison-p4-HW-emulator.json

If you wanna tackle this issue, please leave comment.  
I'll assign to you!

Are you first time for contributing this repo?  
Please read this [guide](https://github.com/TroyTae/game-of-life/blob/master/.github/CONTRIBUTING.md) :)

### PR DESCRIPTION
**Pull request by khabiirk**
resolve #1074

### CODE REVIEW

**Comment by khabiirk** [comment]
Hi, I can't seem to be able to start the dev server even locally. It keeps stopping because of errors.

File: src/life/oscillator/ellison-p4-HW-emulator.json [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
+    [0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 1, 1, 1, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0],
+    [0, 0, 0, 0, 1, 0, 1, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 1, 0, 1, 0, 0, 0, 0],
+    [0, 0, 0, 0, 1, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 1, 0, 0, 0, 0],
+    [0, 1, 1, 0, 1, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 1, 0, 1, 1, 0],
+    [0, 1, 1, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 1, 1, 0],
+    [0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0],
+    [0, 0, 0, 0, 1, 0, 1, 0, 0, 0, 0, 1, 1, 0, 1, 0, 0, 0, 0, 1, 0, 1, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 1, 0, 1, 1, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0],
+    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+
```
Comment (TroyTae) [inline src/life/oscillator/ellison-p4-HW-emulator.json:14]: Please remove this line

**Review by TroyTae — CHANGES_REQUESTED** [review]

**Comment by TroyTae** [comment]
> Hi, I can't seem to be able to start the dev server even locally. It keeps stopping because of errors.

Can you attach the error?

**Comment by khabiirk** [comment]
Screenshot 2020-08-27 at 22 52 45

File: src/life/oscillator/ellison-p4-HW-emulator.json (hunk 1 above)
Comment (khabiirk) [reply src/life/oscillator/ellison-p4-HW-emulator.json:14]: you mean line 13 ?

Comment (TroyTae) [reply src/life/oscillator/ellison-p4-HW-emulator.json:14]: No 14 line, because prettier not allow empty line.

**Comment by TroyTae** [comment]
@khabiirk 
Thank you for finding that bug!
Now I push the fixed commit.

**Review by TroyTae — APPROVED** [review]
LGTM
Thank you for your contribution :)

**Pull request merged**